index.html

```diff
@@ -9,11 +9,87 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+
+<header>
+    
+    
+    <nav id="navbar" class="nav">
+        <img id="logo" src="img/karma-logo.svg" alt=# width="30" height="50" />  
+    
+        
+    <ul class="nav-list">
+      <li>
+        <a href="#welcome-section">meet Karama</a>
+      </li>
+      <li>
+        <a href="#projects">How it works</a>
+      </li>
+      <li>
+        <a href="#contact">stores</a>
+      </li>
+      <a href="#contact">blog</a>
+      </li>
+      <a href="#contact">Help</a>
+      </li>
+      <a href="#contact">Login</a>
+      </li>
+    </ul>
+  </nav>  
+     
+
+</header>
+  <body>
+  <section>
+     <h1>Introduction Karama </h1>
+     <p>  bring wifi with you,every where you go </p>
+     
+     <button >
+     <a href= "#"> learn more  </a>
+     </button>
+ </section>
+ 
+ <div class=icons> 
+     <p>  
+every one needs a little krama</p> </div>
+<div class= icons-img>
+<div class=device>
+    <img src=img/icon-devices.svg
+    alt=""  width="100" height="100" />
+    <p> internet for devices </p>
+     
+</div>
+<div class=coffee>
+    <img src=img/icon-coffee.svg
+    alt=""  width="100" height="100" />
+    <p> Boost yuor productivity </p>
+     
+</div>
+<div class=petrol>
+    <img src=img/icon-refill.svg
+    alt=""  width="100" height="100" />
+    <p> pay as you go </p>
+     
+</div>
+
+
+</div> 
+  
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
 </body>
+<div>
```

Comment:
```suggestion
<footer>
```
It looks like this is a footer, so we can change the element to `footer`. It should also be inside the `body` because all HTML code goes inside the `body`